src/pages/Travel/TravelTerms.tsx

```diff
@@ -35,10 +37,13 @@ function TravelTerms({route}: TravelTermsPageProps) {
 
     useEffect(() => {
         if (travelProvisioning?.error === CONST.TRAVEL.PROVISIONING.ERROR_PERMISSION_DENIED && domain) {
-            handleProvisioningPermissionDeniedError(domain);
+            Navigation.navigate(ROUTES.TRAVEL_DOMAIN_PERMISSION_INFO.getRoute(domain));
+            cleanupTravelProvisioningSession();
         }
         if (travelProvisioning?.spotnanaToken) {
-            openTravelDotAfterProvisioning(travelProvisioning.spotnanaToken);
+            Navigation.closeRHPFlow();
+            cleanupTravelProvisioningSession();
+            Linking.openURL(buildTravelDotURL(travelProvisioning.spotnanaToken));
```

Comment:
Maybe add a comment here to explain why `openURL()` is used instead of `navigate()`

Reply:
done